test_handler.go

```diff
@@ -15,15 +15,16 @@
 package wptdashboard
 
 import (
+    "context"
```

Comment:
Is this in the Go standard lib? When I try running this locally it can't find the package `"context"`.

Reply:
AFAIK It's https://golang.org/pkg/context/, and it's the context.Context interface that gets returned by appengine.NewContext(request)
https://github.com/w3c/wptdashboard/blob/master/test_handler.go#L43

Reply:
Something about both my Go and Python environments is probably majorly screwed up. Just tested this in cloud shell and it works fine.